final-project/README.md

```diff
@@ -5,6 +5,45 @@ This application allows users to sign up for an account, sign in, record and man
 
 The server and client are both written on Go, and run on Docker.
 
+## Project Directory
+```
+final-project/
+│   Taskfile.yml
+│   go.mod
+│   main.go                     - main file for running the server
+│   README.md
+│   routes.go
```

Comment:
Move main.go and routes.go in the "server" folder.
Those files are not really related to the client.